tensorboard/components/plugin_lib/BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:web.bzl", "tf_web_library")
+
+licenses(["notice"])  # Apache 2.0
+
+# TODO(stephanwlee): figure out how this tf_web_library can be used to create
+# maybe a NPM package.
+tf_web_library(
+    name = "plugin_lib",
+    srcs = [
+        "runs.ts",
```

Comment:
Since the host impls are named "runs-host-impl.ts", I'd prefer if "runs.ts" was named something like "runs-guest.ts".  Having both host/guest sources in plugin_lib/ would be less confusing if we could see at a glance, which files will be slurped into the guest library.
Or, wdyt about the pattern of separating out a plugin_lib/guest/ folder?